<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depgraphV::GraphPage</class>
 <widget class="QWidget" name="depgraphV::GraphPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="layoutGroupBox">
        <property name="title">
         <string>Layout</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="layoutAlgorithmLabel">
           <property name="text">
            <string>Algorithm</string>
           </property>
           <property name="buddy">
            <cstring>layoutAlgorithm</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="layoutAlgorithm"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="renderingGroupBox">
        <property name="title">
         <string>Rendering Options</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="highQualityAA">
           <property name="text">
            <string>High Quality AntiAliasing</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="rendererTypeLabel">
           <property name="text">
            <string>Renderer Type</string>
           </property>
           <property name="buddy">
            <cstring>rendererType</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="rendererType">
           <item>
            <property name="text">
             <string>Native</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OpenGL</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="graphAttribsGroupBox">
        <property name="title">
         <string>Graph Attributes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="vertAttribsGroupBox">
        <property name="title">
         <string>Vertices Attributes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="edgeAttribsGroupBox">
        <property name="title">
         <string>Edges Attributes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>46</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="docLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;body&gt;&lt;a href=&quot;http://www.graphviz.org/doc/info/attrs.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;click here for more information&lt;/span&gt;&lt;/a&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>layoutAlgorithm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>onApplyCancel(QAbstractButton*)</slot>
 </slots>
</ui>
